go.mod

```diff
@@ -9,7 +9,8 @@ require (
 	github.com/prometheus/client_golang v1.7.1
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b
+	go.uber.org/goleak v1.1.12
```

Comment:
Going to remove the go.mod changes later, just adding them temporary to run some tests